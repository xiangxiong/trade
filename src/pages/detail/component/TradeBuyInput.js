import React from 'react';
import PropTypes from 'prop-types'
import {TradeBuyInPrice} from '../style';

const TradeBuyInput = (props) =>{
    return (<TradeBuyInPrice>
        <div className="price">{props.label}</div>
        <input className="input" type="text"/>
        <div className="unitWrapper">
            <span className="unit">
            {props.type}
            </span>
        </div>
    </TradeBuyInPrice>);
}

TradeBuyInput.propTypes = {
    label:PropTypes.string.isRequired,
    type:PropTypes.string.isRequired
}

export default TradeBuyInput;
