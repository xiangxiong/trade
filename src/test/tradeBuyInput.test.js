import React from 'react';
import TradeBuyInput from '../pages/detail/component/TradeBuyInput';
import renderer from 'react-test-renderer';

test('Show Input',()=>{
    const component = renderer.create(
        <TradeBuyInput label="test" type="bnb"></TradeBuyInput>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});