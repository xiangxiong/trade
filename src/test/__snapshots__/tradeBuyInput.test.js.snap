// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Show Input 1`] = `
<div
  className="sc-ifAKCX gvOomY"
>
  <div
    className="price"
  >
    test
  </div>
  <input
    className="input"
    type="text"
  />
  <div
    className="unitWrapper"
  >
    <span
      className="unit"
    >
      bnb
    </span>
  </div>
</div>
`;
